### DISCUSSION TITLE
GitHub Education approvals: so many problems

### DISCUSSION THREAD

**Discussion by cmumatt** [community/community#43774] [GitHub Education]
Good afternoon, GitHub,

I applied to renew my GitHub Education account late last year and after a bizarre process and many problems, the automated system finally told me the application would be reviewed within an hour or -- if not -- in five days.

![image](https://user-images.githubusercontent.com/22134678/211401093-acaffec2-aaea-4c17-9e1b-b6a2228c7f66.png)

Well, it's been multiple weeks now.  I opened a ticket in December, but GitHub would never respond and is now auto-closing tickets and dumping academics here so we can... what?  Support ourselves somehow?  We're not the ones who broke this.

Today I tried to re-apply again to see if that might work, and the system told me my status was previously approved.  But there was never any e-mail notification and the ticket I opened never had any activity.  It appears so many things are broken in this new process, including accurate communications.  

The FAQ now says approvals can take "up to six weeks."  So, what are new students in our classes supposed to do for half of the semester?  Just wait patiently until GitHub figures out they are students or bizarrely rejects them so they can start over?  

Ok, so I followed your instructions and posted these problems here, in public.  Thanks!

-Matt

P.S., It takes < five minutes to verify that I'm a student.  My university literally plasters me on its website as a current PhD student.  There are a lot of other people in this discussion board for whom the new system does not appear to be working.

**Comment by illyrius666** [comment]
Yeah i have the problem that it doesn't matter how I fill in my github profile it keeps saying that it doesn't match the image. i used my enrollment document, school portal, receipt from the bursar. it just cannot recognize 2 same names. I am not blind nor dyslexic but clearly the system cannot read.

![image](https://github.com/community/community/assets/28700752/926d68c4-48bf-4676-8d7c-54bd60b86205)

this message for like 13x already. I am getting very frustrated.

**Comment by haitemura** [comment]
Same issue. Whatever I do, my application is always automatically rejected. I applied successfully in 2020 without a problem first try. Now it doesn't work at all. Tried 11 times from different devices, taking photos of my enrolment certificate. I am using my uni mail. Doing everything the application page asks of me.

I always get the following error, even if I do what it says and put in my full name on my profile—which incidentally is a stupidly pointless thing to ask for.

![image](https://github.com/community/community/assets/51244146/0f6fc7cd-a82d-493a-a3fe-d6cccbd19ab5)

**Reply by illyrius666** [reply]
Yes exactly i have to disclose my personal information with the public. to me that sounds very weird to ask for. but yeah it doesn't matter to me anymore since I already did it trying to get past the verification...

**Reply by illyrius666** [reply]
Basically and unfortunately I had to spam it 14x for the system to finally recognize it.

**Reply by haitemura** [reply]
Well, looks like I ought to try some more times then... Did you do anything differently on the application that went through?

**Reply by haitemura** [reply]
My 16th try worked. Things I did differently than before:

- I moved the camera much closer to the text than before, even though now not all of my certificate of enrolment was visible.
- I went into the uni's VPN.

What a horrible experience registering for this.

**Reply by Raja-Debnath** [reply]
I tried 15 times but failed 😭😭😭😭😭

**Comment by StevenLeonCooper** [comment]
Same issue in 2025. GitHub does not even want people using GitHub Education, it's a pathetic joke. As an educator I will be railing against them at every opportunity, my students will not be dealing with this garbage.
